config/installDependencies.py

```diff
@@ -49,15 +49,23 @@
 # The output path can be specified as the first argument if or it defaults
 # to "./dependencies"
 
-platform = "osx" if sys.platform == "darwin" else "linux"
-downloadURL = "https://github.com/GafferHQ/dependencies/releases/download/0.54.0.0/gafferDependencies-0.54.0.0-" + platform + ".tar.gz"
+platform = { "darwin": "osx", "win32": "windows" }.get( sys.platform, "linux" )
+
+if platform == "osx" or platform == "linux" :
+	downloadURL = "https://github.com/GafferHQ/dependencies/releases/download/0.54.0.0/gafferDependencies-0.54.0.0-" + platform + ".tar.gz"
```

Comment:
If you have a sec, be good to bump the linux one to match the windows one whilst you're here.

Reply:
Taken care of in edd8c93